### PR TITLE
Add vllm performance validation doc

### PR DESCRIPTION
**Pull request by peterjunpark**

### CODE REVIEW

File: docs/how-to/performance-validation/mi300x/vllm-benchmark.rst [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+a prebuilt, optimized environment designed for validating large language model
+(LLM) inference performance on the AMD Instinct™ MI300X accelerator. This
+ROCm vLLM Docker image integrates vLLM and PyTorch tailored specifically for the
+MI300X accelerator and includes the following components:
+
+* `ROCm 6.2.1 <https://github.com/ROCm/ROCm>`_
+
+* `vLLM 0.6.4 <https://docs.vllm.ai/en/latest>`_
+
+* `PyTorch 2.5.0 <https://github.com/pytorch/pytorch>`_
```
Comment (peterjunpark) [inline docs/how-to/performance-validation/mi300x/vllm-benchmark.rst:22]: @cindylee2 Updated: vLLM 0.6.4

File: docs/how-to/performance-validation/mi300x/vllm-benchmark.rst [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+      cat /proc/sys/kernel/numa_balancing
+      0
+
+2. Download the :ref:`ROCm vLLM Docker image <vllm-benchmark-unified-docker>`.
+
+   Use the following command to pull the Docker image from Docker Hub.
+
+   .. code-block:: shell
+
+      docker pull rocm/vllm:rocm6.2_mi300_ubuntu20.04_py3.9_vllm_0.6.4
```
Comment (peterjunpark) [inline docs/how-to/performance-validation/mi300x/vllm-benchmark.rst:71]: @cindylee2 Updated: Docker pull commands

Comment (cindylee2) [reply docs/how-to/performance-validation/mi300x/vllm-benchmark.rst:71]: Thank you for updating it.

**Review by cindylee2 — APPROVED** [review]

File: docs/how-to/performance-validation/mi300x/vllm-benchmark.rst (hunk 1 above)
Comment (cindylee2) [reply docs/how-to/performance-validation/mi300x/vllm-benchmark.rst:22]: Thank you

File: docs/how-to/performance-validation/mi300x/vllm-benchmark.rst [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+   .. code-block:: shell
+
+      # disable automatic NUMA balancing
+      sh -c 'echo 0 > /proc/sys/kernel/numa_balancing'
+      # check if NUMA balancing is disabled (returns 0 if disabled)
+      cat /proc/sys/kernel/numa_balancing
+      0
+
+2. Download the :ref:`ROCm vLLM Docker image <vllm-benchmark-unified-docker>`.
```
Comment (amd-jnovotny) [inline docs/how-to/performance-validation/mi300x/vllm-benchmark.rst:65]: Do you want this link to go to Docker or back to the top of this page?

**Review by amd-jnovotny — APPROVED** [review]
looks good

File: docs/how-to/performance-validation/mi300x/vllm-benchmark.rst (hunk 3 above)
Comment (peterjunpark) [reply docs/how-to/performance-validation/mi300x/vllm-benchmark.rst:65]: I wanted it to go the description of the Docker image. The description points to the Docker Hub page. I think it's ok for now.

**Pull request merged**